db/schema.rb

```diff
@@ -13,7 +13,6 @@
 ActiveRecord::Schema.define(version: 2018_11_03_020911) do
 
   # These are extensions that must be enabled in order to support this database
-  enable_extension "pg_stat_statements"
```

Comment:
these just look like noise.

will there be a migration to drop the status table after this change?

Reply:
Yeah. I plan to do that as a separate PR. Just in case anything goes wrong, we'll still have the `statuses` table and all of the `status_id` references.